extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -237,9 +237,18 @@ private static String minusParquetSuffix(@NotNull final String s) {
         return s;
     }
 
-    public static Function<String, String> defaultGroupingFileName(@NotNull final String path) {
-        final String prefix = minusParquetSuffix(path);
-        return columnName -> prefix + "_" + columnName + "_grouping.parquet";
+    /**
+     * Generates grouping file path relative to the table destination file path.
+     *
+     * @param tableDest Destination path for the main table containing these grouping columns
+     * @param columnName Name of the grouping column
+     *
+     * @return The relative grouping file path. For example, for table {@code "A"} with destination
+     *         {@code "dir/A.parquet"} and grouping column {@code "g"}, the method will return
```

Comment:
In the example `{@code "dir/A.parquet"}` , including `dir` is a bit confusing since it isn't actually used.  Or is it?  I think you can just leave that out.

```suggestion
     *         {@code "A.parquet"} and grouping column {@code "g"}, the method will return
```